extensions/cornerstone/src/components/WindowLevelActionMenu/VolumeLighting.tsx

```diff
@@ -60,7 +60,11 @@ export function VolumeLighting({
             min={0}
             type="range"
             step={0.1}
-            style={{ background: calculateBackground(ambient) }}
+            style={{
+              background: calculateBackground(ambient),
+              '--thumb-inner-color': '#5acce6',
+              '--thumb-outer-color': '#090c29',
+            }}
           />
         )}
```

Comment:
oh you are not using our inputRange? weird, can you switch to that? hopefully it is not much work

same for all other files

VolumeLighting, VolumeRenderingQuality and VolumeShift